configstore/doc.go

```diff
@@ -0,0 +1,6 @@
+// Copyright 2022 Fastly, Inc.
+
+// Package configstore provides a read-only interface to config stores. Refer
+// to https://developer.fastly.com/reference/api/dictionaries/ for more
```

Comment:
No new page to link to for "config stores"?

Reply:
https://developer.fastly.com/reference/api/services/resources/config-store/, though this might change.